OneMore/Commands/Tagging/HashtagService.cs

```diff
@@ -211,6 +211,9 @@ protected async Task Scan()
 			if (notebookFilters is not null && notebookFilters.Length > 0)
 			{
 				scanner.SetNotebookFilters(notebookFilters);
+			} else
```

Comment:
This ignores and breaks the "all" notebooks condition. While the UI currently might avoid this case, it was an intentional and possible future case. I want to leave it in.

Reply:
ping